cmd/process-agent/main_common.go

```diff
@@ -27,6 +22,11 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/util/log"
 	"github.com/DataDog/datadog-agent/pkg/util/profiling"
 	"github.com/DataDog/datadog-agent/pkg/version"
+	"github.com/spf13/cobra"
+	"net/http"
+	_ "net/http/pprof"
+	"os"
+	"time"
```

Comment:
nit: the import format seems to be messed up here.

Reply:
Fixed!